mipidsi/src/lib.rs

```diff
@@ -210,26 +211,49 @@ where
         Ok(())
     }
 
+    /// Sets the vertical scroll region.
     ///
-    /// Sets scroll region
-    /// # Arguments
+    /// The `top_fixed_area` and `bottom_fixed_area` arguments can be used to
+    /// define an area on the top and/or bottom of the display which won't be
+    /// affected by scrolling.
+    ///
+    /// Note that this method is not affected by the current display orientation
+    /// and will always scroll vertically relative to the default display
+    /// orientation.
     ///
-    /// * `tfa` - Top fixed area
-    /// * `vsa` - Vertical scrolling area
-    /// * `bfa` - Bottom fixed area
+    /// The combined height of the fixed area must not larger than the
+    /// height of the framebuffer height in the default orientation.
     ///
-    pub fn set_scroll_region(&mut self, tfa: u16, vsa: u16, bfa: u16) -> Result<(), Error> {
-        let vscrdef = dcs::SetScrollArea::new(tfa, vsa, bfa);
+    /// After the scrolling region is defined the [`set_scroll_offset`] can be
+    /// used to scroll the display.
+    pub fn set_vertical_scroll_region(
```

Comment:
I've renamed this and added some more documentation to make it clear that this will only scroll vertically, regardless of the set orientation.